src/platform/netlify.ts

```diff
@@ -0,0 +1,42 @@
+import type PlatformConfigurator from './base';
+import GenericConfig from './generic';
+
+export default class NetlifyConfig extends GenericConfig implements PlatformConfigurator {
+  provider = 'netlify';
+  netlifyBuildId = process.env.BUILD_ID;
+  netlifyContext = process.env.CONTEXT;
+  netlifyDeploymentUrl = process.env.DEPLOY_URL;
+  netlifyDeploymentId = process.env.DEPLOY_ID;
+
+  wrapWebVitalsObject(metrics: any[]) {
+    return [
+      {
+        msg: 'reportWebVitals',
+        webVitals: metrics,
+        _time: new Date().getTime(),
+        platform: {
```

Comment:
This should be `netlify`

Reply:
my idea was that this key is always `platform` and the value of provider is the one that would change, so that we can easily deal with different platforms without having to worry about they key name. so `platform.provider = netlifty` .. etc. what do you think?

Reply:
I think we could also have `platform.provider = vercel`